src/cdk/layout/layout.md

```diff
@@ -2,18 +2,24 @@ The `layout` package provides utilities to build responsive UIs that react to sc
 
 ### BreakpointObserver
 
-`BreakpointObserver`  is a utility for evaluating media queries and reacting to changes in the results of those queries.
+A layout **breakpoint** is viewport size threshold at which a layout shift can occur. The viewport
+size ranges between breakpoints correspond to different standard screen sizes. 
+
+`BreakpointObserver` lets you evaluate media queries to determine the current screen size and
+react to changes when the viewport size changes and crosses a breakpoint.
```

Comment:
Should we mention that the `BreakpointObserver` isn't limited to just screen size viewports? You can use it for things like detecting dark mode or reduced motion.

Reply:
My thinking is that those things are unlikely to change in the middle of the app running, so you could just use `MediaMatcher` directly.